yax/core/src/main/scala/doobie/hi/resultset.scala

```diff
@@ -206,7 +206,7 @@ object resultset {
       val b = Vector.newBuilder[A]
       while (n > 0 && rs.next) {
```

Comment:
`(P)S.setFetchSize(0)` disables cursor so that the driver fetches all results at once. There might be some users who want this behavior, but I'm okay to be opinionated here.